src/org/traccar/protocol/MiniFinderProtocolEncoder.java

```diff
@@ -27,7 +29,7 @@ public String formatValue(String key, Object value) {
         if (key.equals(Command.KEY_ENABLE)) {
             return (Boolean) value ? "1" : "0";
         } else if (key.equals(Command.KEY_TIMEZONE)) {
-            return String.format("%+03d", ((Number) value).longValue() / 3600);
+            return String.format("%+03d", TimeZone.getTimeZone(value.toString()).getRawOffset() / 3600000);
```

Comment:
Why do we need to call `toString()`? Same for other places as well.

Reply:
Because `value` is `Object`. Better use casting?

Reply:
Yes, casting is better if we know it's a String.